docs/specification.md

```diff
@@ -25,11 +25,11 @@ The application is designed to collect and analyze developer activity in Git-bas
 
```

Comment:
The only thing I would leave in this PR is redirecting to the main page on logo click. Other changes I would return because they are not related to the business logic and are not important for the application.